server/testutil.go

```diff
@@ -78,19 +81,24 @@ func NewTestSingleConfig() *Config {
 	cfg.ElectionInterval = typeutil.NewDuration(3 * time.Second)
 	cfg.LeaderPriorityCheckInterval = typeutil.NewDuration(100 * time.Millisecond)
 
-	cfg.Adjust(nil)
+	if err := cfg.Adjust(nil); err != nil {
```

Comment:
This is for test only, can we just assert that it must succeed here?

Reply:
Yes, we can. But I think it's better to check it.

Reply:
Hmmm, so this function will be `func NewTestSingleConfig(c *C)` here, right? Is this a little bit wired since it's in `testutil.go`, not in `*_test.go`？

Reply:
I'm OK with that because it is obviously only used in tests.